pages/hold/confirmation/[id].tsx

```diff
@@ -29,8 +42,14 @@ interface HoldConfirmationPageProps {
  */
 export default function HoldConfirmationPage({
   isEDD = false,
+  pickupLocationLabel,
+  discoveryBibResult,
 }: HoldConfirmationPageProps) {
   const metadataTitle = `Request Confirmation | ${SITE_NAME}`
+
+  const bib = new Bib(discoveryBibResult)
+  const item = bib?.items[0]
+
```

Comment:
We've run into this issue before where models can't be stringified since they contain functions. What we can do is pass just the needed fields but this would slightly limit the flexibility and increase the number of props passed to the component.

The alternative is to remove the model initialization on the server side to avoid running it twice, but we would need to call the discovery response parameters directly which might be fine.

Reply:
You don't need `bib` in this component so you can only return `item`. It would limit to just the current fields you need but you don't need anything else according to the designs (at least for now).